December/09/grace_hopper/info.yaml

```diff
@@ -0,0 +1,13 @@
+title: Grace Hopper
+description: |
+    On December 09, 1906 was born Grace Brewster Murray Hopper.
+    She is an American computer scientist and United States Navy rear admiral.
+    Hopper earned a Ph.D. (1934) in mathematics from Yale and she best known for her trailblazing contributions to computer programming, software development, 
+    and the design and implementation of programming languages.
+    Hopper is credited with popularizing the term "debugging" and "bug", as during her team work on Harvard Mark II computer, her associates discovered a moth that was stuck in a relay 
+    and was impeding the operation of the computer. Upon extraction, the insect was affixed to a log sheet for that day with the notation, "First actual case of a bug being found".
```

Comment:
Lines 7 & 8 are dubious reference at best. We don't have strong support about Hopper actually popularizing these terms and in fact we do have a number of references that the term "bug" was in use by Thomas Edison much earlier.  Given this is mentioned in the "Anecdotes" section of the page I don't feel it matches the purpose of this repository very well.

OTOH in the Standards section of the same page we have a much more fitting contribution: 

> She developed the implementation of standards for testing computer systems and components, most significantly for early programming languages such as FORTRAN and COBOL. The Navy tests for conformance to these standards led to significant convergence among the programming language dialects of the major computer vendors. In the 1980s, these tests (and their official administration) were assumed by the National Bureau of Standards, known today as the National Institute of Standards and Technology.

Reply:
There is reference from national geographic which is more suitable, which confirms this - https://www.nationalgeographic.org/thisday/sep9/worlds-first-computer-bug/

Reply:
> There is reference from national geographic which is more suitable, which confirms this - https://www.nationalgeographic.org/thisday/sep9/worlds-first-computer-bug/

FTR this reference doesn't confirm the statement that "Hopper is credited with popularizing the term "debugging" and "bug"". It suggest that she was probably the person who made the incident with the insect famous.